indico/modules/events/contributions/templates/management/_contribution_list.html

```diff
@@ -79,13 +79,30 @@
                                 {% endif %}
                             </td>
                             <td class="i-table">
-                                <select>
-                                    <option>--No session--</option>
-                                    {% for section in sections %}
-                                        <option>{{ section.name }}</option>
-                                    {% endfor %}
-                                    <option>Add to new session...</option>
-                                </select>
+                                {%- set text_color = '#' + contrib.session.colors.text -%}
+                                {%- set bg_color = '#' + contrib.session.colors.background -%}
+                                <a class="i-button js-session-item-picker"
+                                   style="width: 100%; line-height: 0.9em;color: {{ text_color }} !important;
+                                          background: {{ bg_color }};"
+                                   data-href="{{ url_for('.manage_rest_contrib', contrib) }}"
+                                   data-method="PATCH"
+                                   data-items="{{ sessions | map(attribute='session_data') | list | tojson | forceescape }}"
+                                   data-contrib-id="{{ contrib.id }}"
+                                   data-with-action-button="true"
+                                   data-action-button-title="{% trans %}Add new session{% endtrans %}">
+                                    <span class="label" style="max-width: 80%; overflow: hidden !important;
+                                                               text-overflow: ellipsis; display: inline-block;
+                                                               line-height: 1.2em;">
+                                        {% if contrib.session %}
+                                            {% trans title=contrib.session.title%}
+                                                {{ title }}
+                                            {% endtrans %}
+                                        {% else %}
+                                            {% trans %}Sessions{% endtrans %}
```

Comment:
I don't think we have a strong convention about producing HTML without extra blank lines outside of i18n blocks. I personally don't worry too much about it.